tools/ports/harfbuzz.py

```diff
@@ -48,6 +48,7 @@ def create():
 
     if settings.RELOCATABLE:
       extra_cxx_flags.append('-fPIC')
+      configure_args += ['-DCMAKE_C_FLAGS="-fPIC"']
```

Comment:
Can you rename extra_cxx_flags extra_cflags and then add  it to both CMAKE_CXX_FLAGS and CMAKE_C_FLAGS beelow on line 56.

It seems that it only holds `-fPIC and `-pthread`, neither of which are C++ specific.